### DISCUSSION TITLE
Timeout Exceeded_Dax Studio

### DISCUSSION THREAD

**Discussion by Xavier200492** [DaxStudio/DaxStudio#877] [Q&A] [answered]
Hello everyone! 
I am analyzing my query performance in Dax Studio. When I try running Query Plan and Server Timing I got a message stating that "Timeout exceeded attempting to start Trace. You could try increasing this timeout in the Options". I did this by going to options as it recommended. I redid the step, and got same message. 
May anyone help me out, please?

**Comment by dgosbell** [comment, marked as answer]
What sort of data source are you using? 
Is it AzureAS or the Power BI XMLA Endpoint? 
Are you behind some sort of corporate proxy server? - if so can you try without the proxy, maybe connecting from home or via a mobile hotspot?

**Reply by Xavier200492** [reply]
Thank you in advance for replying my question!
I am using AzureAS. Regarding proxy server, as I am using a corporate computer, I am not allowed to connect to the internet without being linked to VPN from the company.

**Reply by Xavier200492** [reply]
I am also connected to wi-fi from my home (Remote Work)
